skglm/experimental/tests/test_quantile_regression.py

```diff
@@ -28,7 +28,8 @@ def test_PDCD_WS(quantile_level):
     clf = QuantileRegressor(
         quantile=quantile_level,
         alpha=alpha/n_samples,
-        fit_intercept=False
+        fit_intercept=False,
+        solver='highs',
```

Comment:
From v1.4 onward, the default solver is [`highs`](https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.QuantileRegressor.html)

But I think, it is good idea to pin it

Reply:
yes that's weird, I got the issue without specifying. it was quicker to fix this way, even more because it's in a test and not our codebase so I'm ok with not fully understanding...